ui/search/OBASearchResultsMapViewController.m

```diff
@@ -276,11 +272,11 @@ - (void)viewWillAppear:(BOOL)animated {
     [lm addDelegate:self];
     [lm startUpdatingLocation];
 
-    [self refreshSearchToolbar];
-
     if (self.searchController.unfilteredSearch) {
         [self refreshStopsInRegion];
     }
+    
+    self.searchBar.placeholder = [@"Search " stringByAppendingString:[OBAApplication sharedApplication].modelDao.region.regionName];
```

Comment:
if ([OBAApplication sharedApplication].modelDao.region.regionName) {
      self.searchBar.placeholder = [NSString stringWithFormat:NSLocalizedString(@"Search %@",@""), [OBAApplication sharedApplication].modelDao.region.regionName];  
    }
    else {
      self.searchBar.placeholder = NSLocalizedString(@"Search", @"");
    }

for robustness, please.